modules/@apostrophecms/i18n/i18n/en.json

```diff
@@ -45,6 +45,11 @@
   "assetWebpackCacheCleared": "Build cache cleared.",
   "assetWebpackConfigWarning": "⚠️ In the module {{ module }}, your webpack config is incorrect. It must be an object and should contain only the properties: {{ properties }}.",
   "at": "at",
+  "automaticTranslationCheckbox": "Translate text content",
+  "automaticTranslationSettings": "Automatic translation settings",
+  "automaticTranslationSourceErrMsg": "The current locale <strong>{{ source }}</strong> is not suitable for translation. Translation cannot happen.",
+  "automaticTranslationTargetErrMsg": "Could not find a suitable language for the locale <strong>{{ targets }}</strong>. Translation will be skipped when localizing.",
+  "automaticTranslationTargetErrMsg_plural": "Could not find a suitable language for the locales <strong>{{ targets }}</strong>. Translation will be skipped when localizing.",
```

Comment:
What bothers me with those messages (coming from the prototype) is that it's really not clear that the translation will be skipped for the given locales only. At  least this is our plan. Again I think this should be synced with Stu and Tom probably.

Reply:
Updated based on Tom and Stu feedback.